multiclusterhub/charts/grc/templates/grc-rolebinding.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 Red Hat, Inc.
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-grc-rolebinding
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "grc.name" . }}
+    chart: {{ template "grc.chart" . }}
+    component: "ocm-grc-rolebinding"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "grc.name" . }}
+    helm.sh/chart: {{ template "grc.chart" . }}
+roleRef:
+  kind: Role
+  name: {{ .Release.Name }}-grc-role
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: grc-sa
+    
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822